roles/adduser/defaults/main.yml

```diff
@@ -9,13 +9,15 @@ addusers:
     createhome: no
     system: yes
     shell: /sbin/nologin
+    uid: 1000
```

Comment:
system account have uid < 1000 or < 500 depending on the OS, and not sure what happens on upgrade indeed
Also you will never be able to pick a number that work for everyone.
You can add the user creation to your external cert generation script, or fix the permission in kubespray, but this approach is a big NACK for me
/hold

Reply:
I agree that this is not the best solution to the issue. I wanted to get the discussion going around a proper fix though.

I was thinking that moving the cert to a kubernetes secret be a better option. Do you see any drawbacks of this approach?